service/history/replication/executable_task.go

```diff
@@ -273,16 +277,19 @@ func (e *ExecutableTaskImpl) emitFinishMetrics(
 		metrics.OperationTag(e.metricsTag),
 		nsTag,
 	)
-	metrics.ReplicationLatency.With(e.MetricsHandler).Record(
-		now.Sub(e.taskCreationTime),
-		metrics.OperationTag(e.metricsTag),
-		nsTag,
-	)
-	metrics.ReplicationTaskTransmissionLatency.With(e.MetricsHandler).Record(
-		e.taskReceivedTime.Sub(e.taskCreationTime),
-		metrics.OperationTag(e.metricsTag),
-		nsTag,
-	)
+	// replication lag is only meaningful for non-low priority tasks as for low priority task, we may delay processing
+	if e.taskPriority != enumsspb.TASK_PRIORITY_LOW {
```

Comment:
If we omit the lag metrics for low priority tasks, should we at least have some logs when low priority processing is PAUSED/RESUMED?

Reply:
Sure, I will add those log on a separate pr. Besides, metrics on line 275 is still emitted for low priority tasks.